Makefile

```
@@ -25,16 +25,17 @@ build-latest:
 
 build-testing:
 	$(MAKE) build TAG=testing \
-		CORE_REF=v19.13.0 \
-		HORIZON_REF=horizon-v2.26.1 \
-		SOROBAN_RPC_REF=v0.4.0
+		CORE_REF=v20.0.0rc1 \
+		CORE_CONFIGURE_FLAGS='--disable-tests' \
+		HORIZON_REF=horizon-v2.27.0-rc1 \
+		SOROBAN_RPC_REF=v20.0.0-rc2
 
 build-soroban-dev:
 	$(MAKE) build TAG=soroban-dev \
 		CORE_REF=v20.0.0rc1 \
 		CORE_CONFIGURE_FLAGS='--disable-tests' \
```

We don't need to set the core configure flags, as they're set at the top of the file for the defaults.
```suggestion
		HORIZON_REF=horizon-v2.27.0-rc1 \
		SOROBAN_RPC_REF=v20.0.0-rc2

build-soroban-dev:
	$(MAKE) build TAG=soroban-dev \
		CORE_REF=v20.0.0rc1 \
```